packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1725,8 +1725,33 @@ class Transform extends SingleChildRenderObjectWidget {
   /// The origin of the coordinate system (relative to the upper left corner of
   /// this render object) in which to apply the matrix.
   ///
+  /// IMPORTANT: This offset is applied AFTER any [alignment] transformation.
```

Comment:
I would remove "IMPORTANT:" and lower case "AFTER". Just because this kind of emphasis is rarely given in the Flutter docs.